doc/sphinx_source/using/tcl-commands.rst

```diff
@@ -1101,11 +1101,11 @@ onchan <nickname> [channel]
   Module: irc
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-monitor <command> [nickname]
+monitor <sub-command> [nickname]
```

Comment:
can we just write `monitor <add/clear/delete/..> [nickname]` to list all subcommands visible? or perhaps even better list them all under each other so you can find what you're looking for:
```
monitor add <nickname>
monitor status <nickname>
monitor delete <nickname>
monitor clear
```